### PR TITLE
LPS-115878 | LPS-11198 HashGenerator SPI + Message Digest HashGenerator, this pull requires Hash API merged first

### PR DESCRIPTION
**Pull request by arthurchan35**

### CODE REVIEW

File: modules/apps/portal-crypto-hash-generator/portal-crypto-hash-generator-message-digest/src/main/java/com/liferay/portal/crypto/hash/generator/message/digest/MessageDigestHashGenerator.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash-generator/portal-crypto-hash-generator-message-digest/src/main/java/com/liferay/portal/crypto/hash/generator/message/digest/MessageDigestHashGenerator.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/portal-crypto-hash-generator/portal-crypto-hash-generator-message-digest/src/main/java/com/liferay/portal/crypto/hash/generator/message/digest/factory/MessageDigestHashGeneratorFactory.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash-generator/portal-crypto-hash-generator-message-digest/src/main/java/com/liferay/portal/crypto/hash/generator/message/digest/factory/MessageDigestHashGeneratorFactory.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:sf

**Pull request closed without merging**